src/lib/update-alternatives/model/alternative.rb

```diff
@@ -95,6 +104,26 @@ def empty?
       false
     end
 
+    def choice(new_choice_path)
+      unless choices.map(&:path).include?(new_choice_path)
```

Comment:
According to our style guide, we should only use `unless` in its trailing form (that is, at the end of a line) https://github.com/SUSE/style-guides/blob/master/Ruby.md#statements